pkg/fanal/analyzer/pkg/dpkg/copyright.go

```diff
@@ -128,6 +128,11 @@ func (a *dpkgLicenseAnalyzer) Init(opt analyzer.AnalyzerOptions) error {
 }
 
 func (a *dpkgLicenseAnalyzer) Required(filePath string, _ os.FileInfo) bool {
+	// To exclude files from subfolders
```

Comment:
What if using `path.Match(usr/share/doc/*/copyright)`?

Reply:
It works. Thanks. Changed in e519052e68d8bdf4570bfbdf3654102be409d766